test/test-jsdomparser.js

```diff
@@ -6,7 +6,7 @@ var readability = require("../index.js");
 var JSDOMParser = readability.JSDOMParser;
 
 var BASETESTCASE = '<html><body><p>Some text and <a class="someclass" href="#">a link</a></p>' +
-                   '<div id="foo">With a <script>With < fancy " characters in it because' +
+                   '<div id="foo">With a <script>With &lt; fancy " characters in it because' +
```

Comment:
Got it. I'll only replace `<` with `&lt;` in an embedded script there is no CDATA in it.